usagov/vote-gov-drupal
# VOTE-1188 add state string translations

## Jira ticket (required)
[VOTE-1188](https://bixal-projects.atlassian.net/browse/VOTE-1188)

## Description (optional)
Add string translations for state pages

## Deployment and testing (required, if applicable)
### Post-deploy steps
1. run `lando retune`

### Testing steps
1. Visit http://vote-gov.lndo.site/register/ak
2. Switch to all languages and make sure the content of the state page is translated.
